homeassistant/components/garages_amsterdam/binary_sensor.py

```diff
@@ -43,39 +40,22 @@ async def async_setup_entry(
 class GaragesamsterdamBinarySensor(CoordinatorEntity, BinarySensorEntity):
     """Binary Sensor representing garages amsterdam data."""
 
+    _attr_attribution = ATTRIBUTION
+    _attr_device_class = BinarySensorDeviceClass.PROBLEM
+
     def __init__(
         self, coordinator: DataUpdateCoordinator, garage_name: str, info_type: str
     ) -> None:
         """Initialize garages amsterdam binary sensor."""
         super().__init__(coordinator)
-        self._unique_id = f"{garage_name}-{info_type}"
+        self.attr_unique_id = f"{garage_name}-{info_type}"
```

Comment:
Isn't there an `_` missing here? so `self._attr_unique_id = f"{garage_name}-{info_type}"`

Reply:
I think it should be with `_`

Reply:
Thanks. Should be fixed now